js/notify/electron-notify-preload.js

```diff
@@ -113,10 +113,10 @@ function setContents(event, notificationObj) {
     }
 
     // Title
-    titleDoc.innerHTML = notificationObj.title || '';
+    titleDoc.innerText = notificationObj.title || '';
 
     // message
-    messageDoc.innerHTML = notificationObj.text || '';
+    messageDoc.innerText = notificationObj.text || '';
```

Comment:
This isn't going to support any formatted text anymore. The notifications body (message) is only going to support plain text.  Are we okay with that @VishwasShashidhar ?

Reply:
yes, we were removing the HTML tags already, so, this doesn't affect any formatting.